content/en/docs/operations/talos/installation/oci.md

```diff
@@ -0,0 +1,258 @@
+  display_name       = "cozy-dev1-vlan"
+  compartment_id     = var.compartment_id
+  vcn_id             = oci_core_vcn.cozy_dev1.id
+
+  cidr_block         = "192.168.100.0/24"
+  nsg_ids = [oci_core_network_security_group.cozy_dev1_allow_all.id]
+}
+
+variable "node_private_ips" {
+  type    = list(string)
+  default = ["192.168.1.11", "192.168.1.12", "192.168.1.13"]
+}
+
+resource "oci_core_instance" "cozy_dev1_nodes" {
+  count               = length(var.node_private_ips)
+  display_name        = "cozy-dev1-node-${count.index + 1}"
+  compartment_id      = var.compartment_id
+  availability_domain = var.availability_domain
+  shape               = "VM.Standard3.Flex"
+  preserve_boot_volume                        = false
+  preserve_data_volumes_created_at_launch     = false
+
+  create_vnic_details {
+    subnet_id   = oci_core_subnet.test_subnet.id
+    nsg_ids     = [oci_core_network_security_group.cozy_dev1_allow_all.id]
+    private_ip  = var.node_private_ips[count.index]
+  }
+
+  source_details {
+    source_type = "image"
+    source_id   = var.talos_image_id
+  }
+
+  launch_volume_attachments {
+    display_name = "cozy-dev1-node${count.index + 1}-data"
+    launch_create_volume_details {
+      display_name       = "cozy-dev1-node${count.index + 1}-data"
+      compartment_id     = var.compartment_id
+      size_in_gbs        = "512"
+      volume_creation_type = "ATTRIBUTES"
+      vpus_per_gb        = "10"
+    }
+    type = "paravirtualized"
+  }
+
+  shape_config {
+    memory_in_gbs = "32"
+    ocpus         = "4"
+  }
+}
+
+resource "oci_core_vnic_attachment" "cozy_dev1_vlan_vnic" {
+  count       = length(var.node_private_ips)
+  instance_id = oci_core_instance.cozy_dev1_nodes[count.index].id
+
+  create_vnic_details {
+    vlan_id = oci_core_vlan.cozy_dev1_vlan.id
+  }
+}
+```
+
+Apply the configuration using Terraform:
+
+```bash
+oci session authenticate --region us-ashburn-1 --profile-name=DEFAULT
+terraform init
+terraform apply
+```
+
+You'll get three machines with the following IPs:
+- `192.168.1.11`
+- `192.168.1.12`
+- `192.168.1.13`
+
+
+They also have a VLAN interface with the following subnet:
+- `192.168.100.0/24`
+
+This subnet will be used for the internal cluster communication.
+
+## Talos Configuration
+
+Use Talm to apply config and install Talos Linux on the drive.
+
+1. [Download](https://github.com/cozystack/talm/releases/latest) latest Talm binary and save it to `/usr/local/bin/talm`
+2. Make it executable:
+   ```
+   chmod +x /usr/local/bin/talm
+   ```
+
+### Installation with Talm
+
+1. Create directory for new cluster:
+   ```bash
+   mkdir -p mycluster
+   cd mycluster
+   ```
+
+2. Run the following command to initialize Talm for Cozystack:
+
+   ```bash
+   talm init -p cozystack
+   ```
+
+   After initializing, generate a configuration template with the command:
+
+   ```bash
+   talm -n 1.2.3.4 -e 1.2.3.4 template -t templates/controlplane.yaml -i > nodes/nodeN.yaml
```

Comment:
As a new user, I'm losing track here. I *guess* `1.2.3.4` is the public IP of one of my VMs? But what I really don't understand, is *why* this step needs to be done.